packages/platform/src/lib/options.ts

```diff
@@ -13,6 +13,7 @@ export interface PrerenderOptions {
    */
   routes?: string[] | (() => Promise<(string | undefined)[]>);
   sitemap?: SitemapConfig;
+  postRenderingHooks?: ((routes: PrerenderRoute) => Promise<boolean>)[];
```

Comment:
This doesn't need to return anything

```suggestion
  postRenderingHooks?: ((routes: PrerenderRoute) => Promise<void>)[];
```